sdk/attestation/azure-security-attestation/pom.xml

```diff
@@ -0,0 +1,116 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-security-attestation</artifactId>
+  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-sdk-template;current} -->
+
+  <name>Microsoft Azure SDK for Attestation</name>
+  <description>This package contains Microsoft Azure SDK for the Microsoft Azure Attestation service.</description>
+
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
+
+  <scm>
+    <url>https://github.com/Azure/azure-sdk-for-java</url>
+  </scm>
+
+  <dependencies>
+    <dependency>
```

Comment:
These dependencies don't match the ones in our existing dependency version, you'll have to run the python script to update these. The docs are:
https://github.com/Azure/azure-sdk-for-java/blob/master/CONTRIBUTING.md#current-version-dependency-version-unreleased-dependency-version-and-released-beta-dependency-version

The script executed is update_versions.py

Reply:
Ran the scrpit and updated :).